cluster_tools/cluster_tools/executors/multiprocessing_.py

```diff
@@ -113,37 +113,32 @@ def submit(  # type: ignore[override]
         else:
             submit_fn = super().submit  # type: ignore[assignment]
 
-        # Depending on the start_method and output_pickle_path, wrapper functions may need to be
```

Comment:
I believe this comment was not accurate anymore for a while, because we haven't really used the call stack to hold functions and arguments; only the partial functions. Now, I dropped the call stack altogether in favor of partials.

Reply:
I think, the comment is still accurate and the mentioned functions and arguments mean the partial functions. However, the new code using partials should be equivalent as far as I can see. 
cc'ing @daniel-wer who implemented this in https://github.com/scalableminds/webknossos-libs/commit/e773c22ffc86d222a9b57dad77b6494993907955.

Reply:
Also, I think you reversed the order the setup functions are called in. It might be beneficial to setup the logging, first.